### PR TITLE
GOOGLEDOCS-480 : Update Share with GoogleDocs 3.1.0.1

### PR DESCRIPTION
**Pull request by DenisGabriela**

### CODE REVIEW

File: packaging/docker/pom.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
         </dependency>
         <dependency>
             <groupId>org.alfresco.integrations</groupId>
-            <artifactId>alfresco-googledocs-share-community</artifactId>
+            <artifactId>alfresco-googledocs-share</artifactId>
```
Comment (montgolfiere) [inline packaging/docker/pom.xml]: Why do you need to change this ?

File: packaging/docker/pom.xml [hunk 2]
```diff
@@ -119,7 +119,7 @@
                                 </artifactItem>
                                 <artifactItem>
                                     <groupId>org.alfresco.integrations</groupId>
-                                    <artifactId>alfresco-googledocs-share-community</artifactId>
+                                    <artifactId>alfresco-googledocs-share</artifactId>
```
Comment (montgolfiere) [inline packaging/docker/pom.xml]: Why do you need to change this ?

File: packaging/docker/pom.xml (hunk 1 above)
Comment (DenisGabriela) [reply packaging/docker/pom.xml]: I was looking to  https://artifacts.alfresco.com/nexus/content/groups/internal/org/alfresco/integrations/alfresco-googledocs-share/3.1.0.1/ and thought the classifier was removed :). Reverted it. Thanks

File: packaging/docker/pom.xml (hunk 2 above)
Comment (DenisGabriela) [reply packaging/docker/pom.xml]: My mistake, I've reverted it. Thanks.

**Comment by DenisGabriela** [comment]
Green build on Bamboo: https://bamboo.alfresco.com/bamboo/browse/SHARE-SHR6138-2. Ready to merge?

**Comment by montgolfiere** [comment]
✅

**Pull request merged**